Deployment/projects/DeploymentTutorial.json

```
@@ -1,23 +1,57 @@
+    "source" : "client/StoreClient",
     "target" : "topic//my/response/topic"
   }, {
-    "source" : "type/com.vantiq.ims.Store",
-    "target" : "client/DeploymentClient"
+    "source" : "services/com.vantiq.ims.StoreService",
+    "target" : "rule/com.vantiq.ims.StoreService.orderProduct"
   }, {
-    "source" : "type/com.vantiq.ims.Store",
-    "target" : "rule/com.vantiq.ims.checkInventory"
+    "source" : "rule/com.vantiq.ims.StoreService.orderProduct",
+    "target" : "type/com.vantiq.ims.Store"
   }, {
-    "source" : "topic//com.vantiq.ims/NewOrder",
-    "target" : "app/distributeProductApp"
+    "source" : "services/com.vantiq.ims.StoreService",
+    "target" : "type/com.vantiq.ims.Order"
   }, {
-    "source" : "app/distributeProductApp",
-    "target" : "procedure/com.vantiq.ims.shipToStore"
+    "source" : "rule/com.vantiq.ims.StoreService.productReceived",
+    "target" : "services/com.vantiq.ims.StoreService"
+  }, {
+    "source" : "rule/com.vantiq.ims.StoreService.orderProduct",
+    "target" : "services/com.vantiq.ims.StoreService"
+  }, {
+    "source" : "services/com.vantiq.ims.StoreService",
+    "target" : "document/com.vantiq.ims.StoreService.js"
+  }, {
+    "source" : "rule/com.vantiq.ims.StoreService.productReceived",
+    "target" : "type/com.vantiq.ims.Store"
+  }, {
+    "source" : "services/com.vantiq.ims.StoreService",
+    "target" : "rule/com.vantiq.ims.StoreService.productReceived"
+  }, {
+    "source" : "client/StoreClient",
+    "target" : "type/com.vantiq.ims.Store"
+  }, {
+    "source" : "services/com.vantiq.ims.FactoryService",
+    "target" : "document/com.vantiq.ims.FactoryService.js"
+  }, {
+    "source" : "rule/com.vantiq.ims.StoreService.orderProduct",
+    "target" : "services/com.vantiq.ims.FactoryService"
+  }, {
+    "source" : "rule/com.vantiq.ims.FactoryService.requestMoreProducts",
+    "target" : "services/com.vantiq.ims.FactoryService"
+  }, {
+    "source" : "rule/com.vantiq.ims.FactoryService.requestMoreProducts",
+    "target" : "services/com.vantiq.ims.StoreService"
+  }, {
+    "source" : "services/com.vantiq.ims.FactoryService",
+    "target" : "rule/com.vantiq.ims.FactoryService.requestMoreProducts"
+  }, {
+    "source" : "services/com.vantiq.ims.FactoryService",
+    "target" : "type/com.vantiq.ims.Order"
   } ],
   "name" : "DeploymentTutorial",
   "options" : {
-    "description" : "This project is used by the Deployment Tutorial to demo the capability of Deployment Tool.\n\nThis sample application simulates inventory checking on a store and asks the factory for more products when inventory is low. The factory will also notify the store when more products are shipped.",
+    "description" : "This project is used by Deployment Tutotial to demo the capability of Deployment Tool.\n\nThis sample application simulates inventory checking on a store and asks the factory for more products when inventory is low. The factory will also notify the store when more products are shipped.",
```

a few typos:
by -> in the
Tutotial -> Tutorial
demo -> demonstrate
```suggestion
    "description" : "This project is used in the Deployment Tutorial to demonstrate the capability of Deployment Tool.\n\nThis sample application simulates inventory checking on a store and asks the factory for more products when inventory is low. The factory will also notify the store when more products are shipped.",
```